components/loading-spinner/loading-spinner.js

```diff
@@ -0,0 +1,170 @@
+						<circle stroke="none" fill="#000" cx="16" cy="16" r="14"></circle>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-bg" focusable="false">
+					<g fill="none" fill-rule="evenodd" transform="translate(5 5)">
+						<circle stroke-width="0.5" cx="16" cy="16" r="16" class="d2l-loading-spinner-bg-stroke"></circle>
+						<circle stroke="none" fill="#FFF" cx="16" cy="16" r="16"></circle>
+						<circle stroke-width="2" cx="16" cy="16" r="11"></circle>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-slice1" focusable="false">
+					<g fill="none" fill-rule="evenodd">
+						<path d="M24 42h8c0-17.673-14.327-32-32-32v8c1.105 0 2 .895 2 2s-.895 2-2 2v20h20c0-1.105.895-2 2-2s2 .895 2 2z" fill="#FFF"></path>
+						<path d="M0 22c1.105 0 2-.895 2-2s-.895-2-2-2c13.255 0 24 10.745 24 24 0-1.105-.895-2-2-2s-2 .895-2 2c0-11.046-8.954-20-20-20z" fill="#E6EAF0"></path>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-slice2" focusable="false">
+					<g fill="none" fill-rule="evenodd">
+						<path d="M24 42h8c0-17.673-14.327-32-32-32v8c1.105 0 2 .895 2 2s-.895 2-2 2v20h20c0-1.105.895-2 2-2s2 .895 2 2z" fill="#FFF"></path>
+						<path d="M0 22c1.105 0 2-.895 2-2s-.895-2-2-2c13.255 0 24 10.745 24 24 0-1.105-.895-2-2-2s-2 .895-2 2c0-11.046-8.954-20-20-20z" fill="#E6EAF0"></path>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-slice3" focusable="false">
+					<g fill="none" fill-rule="evenodd">
+						<path d="M24 42h8c0-17.673-14.327-32-32-32v8c1.105 0 2 .895 2 2s-.895 2-2 2v20h20c0-1.105.895-2 2-2s2 .895 2 2z" fill="#FFF"></path>
+						<path d="M0 22c1.105 0 2-.895 2-2s-.895-2-2-2c13.255 0 24 10.745 24 24 0-1.105-.895-2-2-2s-2 .895-2 2c0-11.046-8.954-20-20-20z" fill="#E6EAF0"></path>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-slice4" focusable="false">
+					<g fill="none" fill-rule="evenodd">
+						<path d="M24 42h8c0-17.673-14.327-32-32-32v8c1.105 0 2 .895 2 2s-.895 2-2 2v20h20c0-1.105.895-2 2-2s2 .895 2 2z" fill="#FFF"></path>
+						<path d="M0 22c1.105 0 2-.895 2-2s-.895-2-2-2c13.255 0 24 10.745 24 24 0-1.105-.895-2-2-2s-2 .895-2 2c0-11.046-8.954-20-20-20z" fill="#E6EAF0"></path>
+					</g>
+				</svg>
+				<svg viewBox="0 0 42 42" class="d2l-loading-spinner-slice5" focusable="false">
+					<g fill="none" fill-rule="evenodd">
+						<path d="M24 42h8c0-17.673-14.327-32-32-32v8c1.105 0 2 .895 2 2s-.895 2-2 2v20h20c0-1.105.895-2 2-2s2 .895 2 2z" fill="#FFF"></path>
+						<path d="M0 22c1.105 0 2-.895 2-2s-.895-2-2-2c13.255 0 24 10.745 24 24 0-1.105-.895-2-2-2s-2 .895-2 2c0-11.046-8.954-20-20-20z" fill="#E6EAF0"></path>
+					</g>
+				</svg>
+			</div>
+		`;
+	}
+
+	updated(changedProperties) {
+		changedProperties.forEach((oldValue, propName) => {
+			if (propName === 'color') {
+				if (window.ShadyCSS) window.ShadyCSS.styleSubtree(this, {'--d2l-loading-spinner-color': this.color});
```

Comment:
After some digging around, I figured out how to override CSS variables in Lit elements for IE11.  Unlike Polymer elements (which provided an API that wraps shadyCSS), implementors of Lit elements must call shadyCSS `styleSubtree` themselves ([shadyCSS usage doc](https://github.com/webcomponents/polyfills/tree/master/packages/shadycss#usage)).